main/webapp/modules/core/scripts/util/filter-lists-search.js

```diff
@@ -0,0 +1,52 @@
+/*
+* Derived from Portfolio filterable.js by Joel Sutherland
+*
+* Copyright (C) 2009 Joel Sutherland.
+* Licensed under the MIT license
+*
+* Modified for OpenRefine by Simone Povoscania
+* Modified for OpenRefine by Lucero Garcia
+*/
+
+(function($) {
+        $.fn.filterListSearch = function(text) {
+            text = text.trim().toLowerCase();
+            settings = {
+                    selectedTagClass: 'current',
+            }
+            var listElements = $(this);
+            /* FILTER: select a text and filter */
+            listElements.bind("filterSearch", function( e, text ) {
+                // get each project row
+                var projects = $(this).find("tr");
+
+                var projectsWithText = projects.filter(function() {
+                        $(this).removeClass(settings.selectedTagClass);
+                        return $(this).find(".searchable").text().toLowerCase().contains(text)
+                });
+                projectsWithText.addClass(settings.selectedTagClass);
+                $(this).trigger("filterSearchList",[ settings.selectedTagClass ]);
+                var message = $('#no-results-message');
+                if(projectsWithText.length === 0) {
+                    message.show();
+                } else if(message.is(':visible')) {
+                    message.hide();
+                }
+            });
+
+            /* FILTERPORTFOLIO: pass in a class to show, all others will be hidden */
+            listElements.bind("filterSearchList", function( e, classToShow ) {
+                if( text == ''){
+                        return
+                }else{
+                        $(this).trigger("show", [ '.' + classToShow ] );
+                        $(this).trigger("hide", [ ':not(.' + classToShow + ')' ] );
+                }
+            });
+
```

Comment:
```suggestion
                if( text == '') { return; }
                
                $(this).trigger("show", [ '.' + classToShow ] );
                $(this).trigger("hide", [ ':not(.' + classToShow + ')' ] );
            });
```
I see some spacing issues here. All very benine, but still, worth mentionning it.
- no space before and after the `else`.
- this code could be simplified by removing the `else`, since everything after the `if` is already a natural `else`.
- there is a new empty line added after an empty line. I don't see the purpose of that.

Just to make sure, this is not a problem, rather an opportunity to write code easier to read. It is very suggestive.